project-hercules/src/app/calculator/calculator.component.ts

```diff
@@ -24,37 +24,68 @@ import { yellow } from '@scullyio/scully';
           
       ],
   ),
-    // trigger('slideInOut', [
-    //   transition(':enter', [
-    //     style({transform: 'translateY(-100%)'}),
-    //     animate('1800ms ease-in', style({transform: 'translateY(0%)'}))
-    //   ]),
-    //   transition(':leave', [
-    //     animate('1800ms ease-in', style({transform: 'translateY(-100%)'}))
-    //   ])
-    // ])
   ]
   
 })
 
 export class CalculatorComponent implements OnInit {
 
-  showCalculatorForm =  true;
+  showCalculatorForm = true;
   showContactForm = false;
+  showSuccessForm = false;
+  score =  { 
+    "checks": [] as number[],
+    "dataPoints": null as any,
+    "dataAttributes": null as any,
+    "percComplete": null as any,
+    "dataFrequency": null as any,
+    "dataLocation": null as any,
+    "customerEmail": null as any,
+    "customerName": null as any,
+    "customerCompany": null as any,
+    "questions": null as any
```

Comment:
Let's also convert these to their proper types. Looks like most of them are type `string`